src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -8,7 +8,7 @@
     </button>
   </div>
   <div class="modal-body">
-    <img src="assets/Ceph_Logo_Standard_RGB_Reversed_120411_fa.png"
+    <img src="assets/Ceph_Ceph_Logo_with_text_red_white.svg"
```

Comment:
Something is wrong:
![image](https://user-images.githubusercontent.com/399326/91320480-18d21100-e7ad-11ea-8901-30bc555715ac.png)

Reply:
Fixed through Force Push